Add Screens documentation (#2)
SnailyCAD/docs: pages/other/screens.mdx

# Running Snailycad under a screen with linux!

You can run snailycad under a screen so you can safely close the terminal/ssh window without risking it closing out on you!

## Creating a screen

First you need to create the screen do the following:
`screen -S snailycad`
You can change `snailycad` to whatever it's just a name nothing important

> after you enter it, it will immediatly open a screen session here you can goto your snailycad folder nd do 
> `yarn run concurrently "yarn workspace @snailycad/client start" "yarn workspace @snailycad/api generate && yarn workspace @snailycad/api start"`

Reviewer: ```suggestion
after you enter it, it will immediatly open a screen session here you can goto your snailycad folder and run:
 
```bash
yarn run concurrently "yarn workspace @snailycad/client start" "yarn workspace @snailycad/api generate && yarn workspace @snailycad/api start"
```
```

Afterwards do CTRL A + X (DO NOT CTRL C IT WILL CLOSE THE CAD!)

> Now if you want to reopen that session you can list screens by doing

Reviewer: ```suggestion
If you want to reopen that session you can list all the screens using the following command:
```
